hkm/templates/hkm/base.html

```diff
@@ -105,7 +105,7 @@ <h4>{% trans 'Sign up' %}</h4>
     <div class="site-footer">
       <div class="row">
         <div class="col-sm-offset-2 col-sm-4">
-          <img src="/static/hkm/svg/Helsinkikuvia_fi.svg" class="brand-img__footer">
+          <img src="/static/hkm/svg/Helsinkikuvia_fi.svg" class="brand-img-images">
```

Comment:
Minor
Wondering about the name of this class. Looks like it's breaking some sort of BEM syntax. I'm also not sure what "images" is in reference to. It looks like it's applied to a single `<img />` tag.